corehq/apps/notifications/models.py

```diff
@@ -34,7 +36,8 @@ class Meta(object):
         ordering = ["-activated"]
 
     @classmethod
-    def get_by_user(cls, django_user, couch_user, limit=10):
+    def get_by_user(cls, django_user, couch_user, limit=10, plan='Community',
```

Comment:
Adding docstrings for these parameters would be great. In this context, I wouldn't be sure what has_cs_groups is without looking at where this method is called.